server/database/models/article.js

```diff
@@ -0,0 +1,87 @@
+import SequelizeSlugify from 'sequelize-slugify';
+
+export default (sequelize, DataTypes) => {
+  const Article = sequelize.define(
+    'Article',
+    {
+      slug: {
+        type: DataTypes.STRING,
+        unique: true
+      },
+      title: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        validate: {
+          len: {
+            args: [2, 250],
+            msg: 'title must be strings between 2 and 250 chars long'
+          }
+        }
+      },
+      description: {
+        type: DataTypes.STRING(1234),
+        allowNull: true
+      },
+      image: {
```

Comment:
@Adekoreday What is the purpose of the 'image' attribute on the Article model? I feel it would be nicer if you could it more descriptive by specifying if it is a 'featuredImage' or 'previewImage' url of the article.

Reply:
using image actually bound the image as a property of such article for example i feel it would make more sense as article's image than articles featuredImage.

Reply:
i feel the image is okay , as image represents the property of article